pkg/components/velero/component_test.go

```diff
@@ -0,0 +1,193 @@
+// Copyright 2020 The Lokomotive Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package velero //nolint:testpackage
```

Comment:
Actually, we could workaround it by simply exporting `newComponent()` function. Not very consistent with other components, but I think we would need to export it anyway.

Reply:
Seems a bit out of scope for this PR if we are considering for all components. If we do it just for this component then we need to take care of others as well.

Reply:
I'm open to change it for just this component and handle others later, as part of the dedicated issue, as I prefer not repeating bad habits over consistency, but it's open for discussion.

Reply:
I'll create a new PR targeting all components and the scope of this out from this PR.